__tests__/app.test.js

```diff
@@ -114,6 +114,61 @@ describe("App", () => {
           expect(response.body.msg).toBe("Bad request");
         });
     });
+    test("PATCH:200 updates an article by article_id and responds with the updated article", () => {
+      const updatedArticle = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: -1,
+        article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      }
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: -101 })
+        .expect(200)
+        .then((response) => {
+          expect(response.body.article).toEqual(updatedArticle);
+        });
+    });
+    test('PATCH:400 responds with error message when passed invalid data', () => {
+      return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 'one' })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+    });
+    test('PATCH:400 responds with error message when passed no data', () => {
+      return request(app)
+      .patch("/api/articles/1")
+      .send({})
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+    });
+    test("PATCH:400 responds with error message when given an invalid id", () => {
```

Comment:
This test is a little different and I'm not sure you're getting the 400 you want here. In the 200 tests you're sending the inc_votes as a number. In this one you're sending a string of "10" which should be an error all of its own.